### PR TITLE
[#858] Fix typos. The file says that it is generated code, but I could find the files which were used as source. Please update the source files.

### PR DESCRIPTION
**Pull request by RubenGarcia**
The file says that it is generated code, but I could find the files which were used as source. Please update the source files.

The lib/api/README.txt file indicates "All functions are automatically generated by the apiGen script in the scripts directory."
But I could not find such a script in the respository.

### CODE REVIEW

**Comment by RubenGarcia** [comment]
It would probably be good to update lib/api/README.txt with the location of the mentioned script.

File: lib/api/src/rcBulkDataObjPut.cpp [hunk 1]
```diff
@@ -113,9 +113,9 @@
  *         are:
  *    \n COL_DATA_NAME - the full paths of each data object.
  *    \n COL_DATA_MODE - the createMode of each data object.
- *    \n OFFSET_INX - The accummulative size of the data objects. e.g.,
+ *    \n OFFSET_INX - The accummulative size of the data objects. E.g.,
```
Comment (trel) [inline lib/api/src/rcBulkDataObjPut.cpp:116]: "accummulative" -> "cumulative"

**Review by trel — COMMENTED** [review]
please put "[#858] " at the front of your commit message and we'll get it merged.

thanks!

**Comment by RubenGarcia** [comment]
Added your last change request and changed the commit message as requested.

**Comment by trel** [comment]
oops, didn't notice it was two commits.   we goofed.   we'll do better next time.

**Pull request merged**